wallet/src/de/schildbach/wallet/ui/dashpay/utils/DashPayConfig.kt

```diff
@@ -34,11 +39,12 @@ open class DashPayConfig @Inject constructor(private val context: Context) {
     companion object {
         const val DISABLE_NOTIFICATIONS: Long = -1
 
+        const val DASHPAY_PREFS_DIRECTORY = "dashpay"
         val LAST_SEEN_NOTIFICATION_TIME = longPreferencesKey("last_seen_notification_time")
         val LAST_METADATA_PUSH = longPreferencesKey("last_metadata_push")
+        val HAS_DASH_PAY_INFO_SCREEN_BEEN_SHOWN = booleanPreferencesKey("has_dash_pay_info_screen_been_shown")
     }
-
-    private val Context.dataStore by preferencesDataStore("dashpay", produceMigrations = {
+    private val Context.dataStore by preferencesDataStore(DASHPAY_PREFS_DIRECTORY, produceMigrations = {
```

Comment:
This isn't a directory. It's the name of the preference file; all files are stored in the same directory.